docs/resources/control-flow/functions/action-blocks.md

```diff
@@ -16,6 +16,7 @@ Action Blocks have different scopes, which determine their availability:
 | **Page Level Action Blocks**     | Restricted to the page in which they were created. These can access the state variables available in their scope, such as page state variables, as well as variables above their scope, such as [App State variables](../../../resources/data-representation/app-state.md). | Page Level Action Blocks can access page state variables and App State variables.                                                                              |
 | **Component Level Action Blocks**| Restricted to the component in which they were created. These can access the state variables available in their scope, such as component state variables, as well as variables from higher scopes, like page and App State variables. | Component Level Action Blocks can access component state variables, page state variables, and App State variables.                                             |
 
+> :information_source: **Note**: Some actions are not supported and cannot be used in an Action Block, these actions are hidden there by default.
```

Comment:
Can we add 1-2 example Actions here that are not available in Action Blocks to provide more clarity?

Reply:
Noted.